ui/src/pages/users.vue

```diff
@@ -512,6 +705,23 @@ fetch_all_users();
 .usertable {
   --va-data-table-cell-padding: 3px;
 }
+
+.trash-can-button-text {
+  color: var(--va-primary);
+  font-size: 12px;
+  font-weight: 600;
+  flex: none;
+}
+
+.deletion-type-label {
+  font-size: 1rem;
+  font-weight: bold;
+}
+
+.deletion-options {
+  display: flex;
+  gap: 10px; /* Space between radio buttons */
+}
```

Comment:
These two CSS styles can be applied through the `class` prop. I think something like `class="flex gap-10"` should do it. Tailwind flex/gap docs - https://tailwindcss.com/docs/flex and  https://tailwindcss.com/docs/gap